scripts/create_AR_DeployInst.py

```diff
@@ -408,6 +456,7 @@ def determine_deployment_schedule(
         if value <= 0:
             continue
         # redeploy reactors
+        num_rxs = {k:v for k, v in zip(reactors, np.repeat(0,len(reactors)))}
```

Comment:
```suggestion
        num_rxs = {k:v for k, v in zip(reactors, np.repeat(0,len(deployment_order)))}
```

For clarity and consistency with my previous suggestions, the `reactor` variable here should be renamed to something like `reactor_order` or `deployment_order`. 

Also, you can replace the condition

```py
if shares is not None:
# is equivalent to
if shares:
```
`if None` returns `False` in Python.